### DISCUSSION TITLE
Is places.sources.confidence deprecated

### DISCUSSION THREAD

**Discussion by lilcorey** [OvertureMaps/data#330] [Discussion]
Every record of the GA release shows NULL confidence score. Is this correct or is my query wrong? Does anyone have context for why it isn't populated anymore?

```
select array_extract(sources, 1)
from read_parquet('overture/places_2024_07_22/*', filename=true) 
where array_extract(sources, 1)['confidence'] is not null
```

**Comment by jwass** [comment]
Hi @lilcorey - the 'place' type still has a top-level column called "confidence" https://github.com/OvertureMaps/schema/blob/main/schema/places/place.yaml#L44 - which is still populated in the latest release: In Athena: `select count(1) from overture.release.v2024_07_22_0 where type = 'place' and confidence is not null` returns 53289793 rows.

I agree that the relationship between this column and the confidence in the individual sources is somewhat confusing, but this top-level column allows an overall assessment of the confidence from all sources
